src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -32,5 +32,13 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
     public Optional<List<? extends ReadOnlyPerson>> getRelevantPersons() {
         return Optional.ofNullable(relevantPersons);
     }
+    
+    /**
+     * get feedbackToUser member
+     * @return feedbacktoUser
+     */
```

Comment:
pls remove auto gen header comments if they are not useful. Usually accessors don't need comments as their names are informative enough.